src/screens/AddNewItemScreen/index.tsx

```diff
@@ -26,6 +27,16 @@ export const AddNewItemScreen: FC = () => {
   const currentDate = `${date.getDate()}.${date.getMonth()}.${date.getFullYear()} at ${date.getHours()}:${date.getMinutes()}`;
   const buttonStyle = text ? style.button : style.buttonDis;
 
+  console.log('screen text', text);
+
+  useEffect(() => {
+    navigation.setOptions({
+      headerLeft: () => {
+        return <AddNewItemScreenNavigationHeader itemText={text} />;
+      },
+    });
+  }, [text, navigation]);
```

Comment:
I think it's no good idea to set new button every time when text changes

we should provide function from closure and this function should has access to current text